helm-util/helm.libsonnet

```diff
@@ -0,0 +1,57 @@
+local d = import 'github.com/sh0rez/docsonnet/doc-util';
+{
+  local this = self,
+
+  '#': d.pkg(
+    name='helm-util',
+    url='github.com/grafana/jsonnet-libs/helm-util/helm.libsonnet',
+    help='`helm-util` provides utilities for using helm in jsonnet',
+  ),
+
+  '#helmTemplate': d.fn(
+    '`helmTemplate` wraps the helmTemplate native function and returns a rendered chart as a map of kubernetes manifests',
```

Comment:
As a user, I don't quite care about the native func. I want to know what happens when I call this. Maybe like so?

```markdown
`template` expands the Helm Chart to it's underlying resources and returns them in an `Object`,
so they can be consumed and modified from within Jsonnet.

This functionality requires Helmraiser support in Jsonnet (e.g. using Grafana Tanka) and also
the `helm` binary installed on your `$PATH`.
```